### PR TITLE
Read and deserialize actual protobufs

### PR DESCRIPTION
**Pull request by arielshaqed**
Also transition to `mapreduce.InputFormat` rather than `mapred.InputFormat` (it's better because it puts the "uce" into "mapreduce", and also  https://stackoverflow.com/questions/16269922/hadoop-mapred-vs-hadoop-mapreduce.

### CODE REVIEW

File: src/main/scala/io/treeverse/clients/LakeFSInputFormat.scala [hunk 1]
```diff
@@ -1,68 +1,95 @@
...
-    localFile.delete()
     // TODO(johnnyaug) ask lakeFS metadata server for the sstable url
-    ranges.map(r => new FileSplit(new Path(p.getParent + "/" + new String(r.identity)), 0, 0, null.asInstanceOf[Array[String]])).toArray
+    val rangesReader = new SSTableReader(localFile.getAbsolutePath, RangeData.newBuilder().build())
+    localFile.delete()
+
+    val ranges = read(rangesReader)
+
+    ranges.map(
+      r => new FileSplit(new Path(p.getParent + "/" + new String(r.id)), 0, 0, null.asInstanceOf[Array[String]])
```
Comment (johnnyaug) [inline src/main/scala/io/treeverse/clients/LakeFSInputFormat.scala]: Should use GravelerSplit here?

**Review by johnnyaug — CHANGES_REQUESTED** [review]

File: src/main/scala/io/treeverse/clients/LakeFSInputFormat.scala (hunk 1 above)
Comment (arielshaqed) [reply src/main/scala/io/treeverse/clients/LakeFSInputFormat.scala]: Wow, "you had one job"...

**Review by arielshaqed — COMMENTED** [review]
Yikes!  Thanks...

**Comment by arielshaqed** [comment]
Abandoning this in favour of a new PR -- one that actually works.  (Serialize GravelerSplit, add some configurability, write a test program, etc.).

**Pull request closed without merging**